library/src/webapp-filtered/editor/ckeditor.launch.js

```diff
@@ -150,6 +150,15 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
         };
     }
 
+    let siteId = "";
+    if (collectionId) {
+        if (collectionId.startsWith('/user/') && typeof portal !== 'undefined' && portal.siteId !== '') {
```

Comment:
You can go:
if (collectionId.startsWith('/user/') && portal && portal.siteId) {

You could risk optional chaining, too, but mobile support is recent.
https://caniuse.com/mdn-javascript_operators_optional_chaining